trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.ts

```diff
@@ -0,0 +1,31 @@
+// "Recommended Security and Performance Max: 2048 CHARACTERS" - https://stackoverflow.com/a/48230425
+const MAX_URL_CHARACTERS = 2048;
+const BASE_URL_CHARACTER_ALLOWANCE = 150;
+
+export const getAddToExistingPaymentBackLinkToUtilisationReportReconciliationPage = (reportId: string, feeRecordIds: number[]): string => {
+  const urlWithoutParams = `/utilisation-reports/${reportId}`;
+
+  if (feeRecordIds.length === 0) {
+    return urlWithoutParams;
+  }
+
+  const commaSeparatedFeeRecordIds = feeRecordIds.join(',');
+
+  const urlParams = new URLSearchParams({
+    selectedFeeRecordIds: commaSeparatedFeeRecordIds,
+  }).toString();
+
+  const urlWithParams = `${urlWithoutParams}?${urlParams}`;
+
+  const urlWithParamsMaxLength = MAX_URL_CHARACTERS - BASE_URL_CHARACTER_ALLOWANCE;
```

Comment:
Feels a bit strange to have 2x constants if we're just using them to create a new max length, could it just be one constant? Or is there a vision to use both constants later?

Reply:
It's more of trying to make it clear how the max length is reached - as we don't know the length of the base URL we're adding an allowance for this and I feel as though it's a bit clearer as to how we reach the max length value of  1898 chars (2048 - 150) by having both the constants vs putting all this in a comment.

Reply:
Related to the comment here: https://github.com/UK-Export-Finance/dtfs2/pull/3399/files/9a6775cdfb338b30bd09a77b9bc487ee456f7780#r1733056912

Think this thread should be resolved given that comment